app/src/styles.sass

```diff
@@ -29,7 +29,7 @@ input
 .formactions
   padding: $small-space 0
   display: flex
-  justify-content: flex-end
+  justify-content: center
```

Comment:
Why have you changed that? This is a very big change, impacting the whole application. See for instance:
![image](https://user-images.githubusercontent.com/1835665/119094581-87c55900-ba11-11eb-9ed1-960fa79e7f38.png)
It would have been better to propose this change before implementing it.

Reply:
Sorry Sir, I didn't realise that the style of button applies on other pages. I just considered about the page of "inscription" and the "login". It is normal to have the submit bottons at center for these 2 pages. I will propose my changes on slack before my modifications. Sorry for the inconvenient.

Reply:
Please don't be sorry, and don't call me Sir.

Reply:
okay haha. I want to center the button because the design that I have seen is very often at the center. For example, the ENT of UT1, the login button is at the center. So for me it is just we have the habite of having the submit buttons at center.

Reply:
Ok. The buttons was on the right for the sake of consistency across the whole application. If you really want these buttons to be centered then, to not break consistency, we could make them spawn the width of their enclosing box. In that case, you'll have to modify the html file for them to not be inside div.formactions anymore.